### ISSUE TITLE
add implementation of a simple model?

### ISSUE THREAD

**Issue by amueller** [amueller/scipy-2016-sklearn#23]
We discussed adding linear regression with gradient descent in the beginning. Do we still want to do that?

**Comment by rasbt** [comment]
I can do that now! What about appending it to "11 Review of Scikit-learn API.ipynb"?

**Comment by amueller** [comment]
If you like, sure. We probably have wayyy too much material, so we need to see where we can cut it. Depends a bit on how much time we spend on exercises, though. We definitely need to check whether all our exercises make sense.

**Comment by rasbt** [comment]
Okay let's cut it out then, I think you are right, I really have no feeling for how long the notebooks/exercises take, you are much better at judging :)
